src/Organisms/NotFoundPage/NotFound.js

```diff
@@ -0,0 +1,23 @@
+import notFound from '../../Assets/notFound.png';
```

Comment:
I think, this folder could have different name? For pages we have Pages, and name suggested it is whole page. Maybe NotFoundContent? Or something like that?

Reply:
Ok, earlier it was in pages, so that is a problem ;) But I'll change it :)